reference/constraints/Choice.rst

```diff
@@ -367,10 +367,13 @@ too many options per the `max`_ option.
 strict
 ~~~~~~
 
-**type**: ``boolean`` **default**: ``false``
+**type**: ``boolean`` **default**: ``true``
+
+The validator will also check the type of the input value.
```

Comment:
Wouldn't it make more sense to keep the current description? Or at least tell a bit about what "check the type" means?